src/pages/AddPersonalBankAccountPage.js

```diff
@@ -49,82 +46,44 @@ class AddPersonalBankAccountPage extends React.Component {
     constructor(props) {
         super(props);
 
-        this.getErrorText = this.getErrorText.bind(this);
-        this.clearError = this.clearError.bind(this);
         this.validate = this.validate.bind(this);
         this.submit = this.submit.bind(this);
 
         this.state = {
             selectedPlaidBankAccount: undefined,
-            password: '',
         };
-
-        this.formHelper = new FormHelper({
-            errorPath: 'personalBankAccount.errorFields',
-            setErrors: errorFields => ReimbursementAccount.setPersonalBankAccountFormValidationErrorFields(errorFields),
-        });
     }
 
     componentDidMount() {
         BankAccounts.clearPersonalBankAccount();
     }
 
     /**
-     * @returns {Object}
-     */
-    getErrors() {
-        return this.formHelper.getErrors(this.props);
-    }
-
-    /**
-     * @param {String} fieldName
-     * @returns {String}
-     */
-    getErrorText(fieldName) {
-        const errors = this.getErrors();
-        if (!errors[fieldName]) {
-            return '';
-        }
-
-        return this.props.translate(this.errorTranslationKeys[fieldName]);
-    }
-
-    /**
-     * @param {String} path
-     */
-    clearError(path) {
-        this.formHelper.clearError(this.props, path);
-    }
-
-    /**
-     * @returns {Boolean}
+     * @param {Object} values - form input values passed by the Form component
+     * @returns {Ojbect}
      */
-    validate() {
+    validate(values) {
         const errors = {};
         if (_.isUndefined(this.state.selectedPlaidBankAccount)) {
             errors.selectedBank = true;
         }
 
-        if (this.props.isPasswordRequired && _.isEmpty(this.state.password)) {
+        if (this.props.isPasswordRequired && _.isEmpty(values.password)) {
             errors.password = true;
```

Comment:
Looks like `isPasswordRequired ` is `undefined`, so code inside will never run. I think we only wanna show an error if the pasword is empty? Display something like "This field is required"? Or is the backend error enough?

Reply:
Oh if we don't require a password to add a personal bank account, we don't need to validate it in the front-end either. We could just remove this check.

Reply:
I think it is required to add a personal bank account. But for some reason we never set `isPasswordRequired` to true. Because if you send an empty password the backend sends back an "invalid password" error. I can either remove the check and rely on the backend, or add a check to validate the field is not empty.

Reply:
The password field is required, so I think that we can remove mentions to that prop and:
1. Add a check for empty password
2. Add check for [invalid password](https://github.com/Expensify/App/blob/ce740fedbcd9a8b334d3fb8e36bce76a542cb577/src/libs/ValidationUtils.js#L272-L274)

Reply:
> 2. Add check for [invalid password]

I think we're asking the user there to input their existing password? So it should already be valid, unless it's incorrect then the backend will handle that case. What do you think?

Reply:
I think that's fine too. I don't have strong feelings about 2. We should still update the validate method to account for an empty password and remove the check for `selectedPlaidBankAccount`

Reply:
@youssef-lr any updates regarding my comment above?

Reply:
On it, sorry! Was OOO (sick).